Apps/W1/HybridSL/app/src/pages/CloudMigrationMgmtExt.PageExt.al

```diff
@@ -0,0 +1,230 @@
+
+                        trigger OnDrillDown()
+                        begin
+                            Page.Run(Page::"SL Migration Warnings");
+                        end;
+                    }
+                    field("SL Cloud Migration Errors"; MigrationErrorCount)
+                    {
+                        ApplicationArea = All;
+                        Caption = 'SL Cloud Migration Errors';
+                        ToolTip = 'Indicates the number of SL migration error entries.';
+
+                        trigger OnDrillDown()
+                        begin
+                            Page.Run(Page::"SL Migration Error Overview");
+                        end;
+                    }
+                    field("SL Posting Errors"; PostingErrorCount)
+                    {
+                        ApplicationArea = All;
+                        Caption = 'Posting Errors';
+                        Style = Unfavorable;
+                        StyleExpr = (PostingErrorCount > 0);
+                        ToolTip = 'Indicates the number of posting errors that occurred during the migration.';
+
+                        trigger OnDrillDown()
+                        begin
+                            Page.Run(Page::"SL Migration Warnings");
+                        end;
+                    }
+                    field("SL Failed Companies"; FailedCompanyCount)
+                    {
+                        ApplicationArea = Basic, Suite;
+                        Caption = 'Failed Companies';
+                        Style = Unfavorable;
+                        StyleExpr = (FailedCompanyCount > 0);
+                        ToolTip = 'Indicates the number of companies that failed to upgrade.';
+
+                        trigger OnDrillDown()
+                        begin
+                            Page.Run(Page::"SL Hybrid Failed Companies");
+                        end;
+                    }
+                    field("SL Failed Batches"; FailedBatchCount)
+                    {
+                        ApplicationArea = All;
+                        Caption = 'Failed Batches';
+                        Style = Unfavorable;
+                        StyleExpr = (FailedBatchCount > 0);
+                        ToolTip = 'Indicates the total number of failed batches, for all migrated companies.';
+
+                        trigger OnDrillDown()
+                        begin
+                            Message(FailedBatchMsg);
+                        end;
+                    }
+                }
+            }
+        }
+    }
+
+    actions
+    {
+        addafter(RunReplicationNow)
+        {
+            action(SLConfigureMigration)
+            {
+                ApplicationArea = All;
+                Caption = 'Configure SL Migration';
+                Enabled = HasCompletedSetupWizard;
+                Image = Setup;
+                Promoted = true;
```

Comment:
Run the action to move the actions to the new promoted syntax. It is a code action that can do this automatically.